.github/workflows/tests.yml

```diff
@@ -66,18 +75,38 @@ jobs:
               apt-get update
               apt-get install -y git libc6:amd64 libstdc++6:amd64
 
-          - name: Ubuntu Focal (20.04)
-            image: ubuntu:focal
+          - name: Debian Testing
```

Comment:
If you want to avoid inflating the number of parallel builds, you could combine this with one of the build variants above.

Reply:
Good point. I don't feel strongly here one way or the other.